src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -52,6 +52,5 @@ target_link_libraries(
   EvolveGhKerrSchildNumericInitialData
   PRIVATE
   GeneralRelativitySolutions
-  INTERFACE
```

Comment:
I think the squashing went wrong here, this shouldn't be here in the first place

Reply:
Maybe, but is there a change you actually want me to make here? The line shouldn't be there, and this is the only commit in my PR that touches it

Reply:
If I'm not mistaken this line is added in the commit "Move IO/Importers to separate library" and removed again here

Reply:
Thanks! I edited the commit to remove the spurious line